.github/workflows/k3d-verify-warden-manifest.yaml

```diff
@@ -0,0 +1,30 @@
+name: k3d integration test
+
+on:
+  push:
+    branches: [ "main", "release-*" ]
+  workflow_dispatch:
+  pull_request:
```

Comment:
`pull_request_target` might be more secure (if it will work in our context) https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target